README.md

```diff
@@ -84,6 +102,8 @@ cmake ../
 make
 [sudo] make install
 ```
+If using an environment, remember to set `CMAKE_INSTALL_PREFIX` to the value of `$CONDA_PREFIX`.
+
 Notice: `sudo` is not necessary if you specify the `CMAKE_INSTALL_PREFIX`. In this case it is necessary to add in the `.bashrc` or `.bash_profile` the following lines:
```

Comment:
What do you mean?

Reply:
Sorry my comment was originally pointing to the line below,  [107](https://github.com/robotology/walking-controllers/pull/167/files/ead5b3190d02465c57823a660506f232bf34f783#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R107). It is now ambiguous since the `CMAKE_INSTALL_PREFIX` is updated also with the conda environment but in such case there is no need to add to the `.bashrc` the environment variables listed. 

Maybe it can be rephrased as: "Notice: if you're using the `robotology-superbuild`, you don't need to use sudo if you specify the CMAKE_INSTALL_PREFIX. In such case, it's essential to include the following lines in either the .bashrc or .bash_profile file:"